_layouts/partners/roadmap.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: base
+---
+<header>
+  {% include skip_nav.html %}
+  {% include banner.html %}
+  {% include partners/header.html %}
+</header>
+<main id="main-content">
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <h1 class="tablet:grid-col-6 partners-title">{{ page.title }}</h1>
+        <div class="tablet:grid-col-12 partners-border"></div>
+        <div class="roadmap-body grid-row">
+          <div class="text tablet:grid-col-7 grid-col-12">
+            <div>{{ page.subtitle | markdownify}}</div>
+            <div class="partners-subtitle">{{ page.subsection | markdownify }}</div>
+          </div>
+          <div class="image tablet:grid-col-5 grid-col-12">
+            <a aria-label="Program Roadmap" href="{{ site.baseurl }}{{ page.roadmap-url }}"><img class="section-title-right" alt="Program Roadmap" src="{{ site.baseurl }}/assets/img/partners/roadmap.png"/></a>
```

Comment:
I don't believe you'd need the `aria-label` if the image has an equivalent `alt` text.

```suggestion
            <a href="{{ site.baseurl }}{{ page.roadmap-url }}"><img class="section-title-right" alt="Program Roadmap" src="{{ site.baseurl }}/assets/img/partners/roadmap.png"/></a>
```

Reply:
Also, as far as the text alternative itself, maybe it could be more accurate to describe it as a screen grab of the intro slide of the deck itself.

e.g.

```suggestion
            <a href="{{ site.baseurl }}{{ page.roadmap-url }}"><img class="section-title-right" alt="Program Roadmap intro slide" src="{{ site.baseurl }}/assets/img/partners/roadmap.png"/></a>
```

Reply:
Sounds good to me, edited.